webcompat/static/css/development/main.css

```diff
@@ -51,3 +51,4 @@
 @import "components/label";
 @import "components/upload-form";
 @import "components/contributors";
+@import "components/Markdown";
```

Comment:
Maybe Travis doesn't like `Markdown` vs `markdown` here? (I guess Linux cares about case sensitivity and OSX doesn't? :confused:)